src/YesSql.Core/Session.cs

```diff
@@ -780,17 +784,19 @@ private void BatchCommands()
 
             if (createDocumentCommands != null)
             {
-                foreach (var page in createDocumentCommands.PagesOf(_store.Configuration.CommandsPageSize))
+                foreach (var page in createDocumentCommands.PagesOf(_store.Configuration.CommandsPageSize)
+                    .SplitPagesBy(c=>c.CollectionName))
```

Comment:
This call to SplitPagesBy is one thing I don't feel pretty well.
The poins is I saw that you pass to dapper an string with an insert in the pseudo SQL and a collection of documents to insert. Problem I found is the SQL is not the same for all the documents because some of them are for one collection and others are for others one.
I had two options: 

- To group inserts (or updates on next foreach) to make as much as possible inserts in one batch commad. But if generates me doubts cause I'm not sure it is right to change the oreder on which things are inserted on db.
- To split each page every time I find a document with a different collection than previous one. Problem is this breaks  batch in small pieces.

What do you think, is it ok like it is?
Thank you